fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -934,18 +934,13 @@ private void transferToLeader() {
             if (!haProtocol.fencing()) {
                 throw new Exception("fencing failed. will exit");
             }
-            long replayStartTime = System.currentTimeMillis();
-            // replay journals. -1 means replay all the journals larger than current journal id.
-            replayJournal(JournalCursor.CUROSR_END_KEY);
-            long replayEndTime = System.currentTimeMillis();
-            LOG.info("finish replay in " + (replayEndTime - replayStartTime) + " msec");
-
+            long maxJournalId = journal.getMaxJournalId();
```

Comment:
Can you describe what enhancement will this pr do? Is it more stable to replay to a specified key than to replay to END_KEY?

Reply:
No, as the title, it add a extra procedure of checking replayed journal id in transferToLeader. You can see that in line 1673-1682 of this file. To check the id, we have to use a definite key instead of END_KEY, this is just a side effect.
Also it fix some logs, change the word master to leader.